### PR TITLE
Maint/master/update removal tickets

### PR DESCRIPTION
**Pull request by melissa**

### CODE REVIEW

**Comment by shermdog** [comment]
:+1:

File: tasks/platform-removal.rake [hunk 1]
```diff
@@ -66,6 +81,9 @@ DOC
 
   description[:graphite_vmpooler] = <<-DOC
 Remove platform from graphite
+
+**NOTE**
+This task is optional. Graphite will automatically clears after 2 weeks.
```
Comment (underscorgan) [inline tasks/platform-removal.rake]: 'Will automatically clear' or 'clears'?

File: tasks/platform-removal.rake [hunk 2]
```diff
@@ -46,6 +56,11 @@ DOC
 Remove platform definition and any special tweaks from puppet-agent https://github.com/puppetlabs/puppet-agent/tree/master/configs/platforms
 DOC
 
+  description[:puppet_agent_whitelist] = <<-DOC
+Update either the foss_platforms or pe_platforms list in puppet-agent ext/build_defaults.yaml so it can be properly whitelisted for nightly builds.
+DOC
```
Comment (underscorgan) [inline tasks/platform-removal.rake]: Probably 'properly removed from whitelisting for nightly builds'?

Comment (melissa) [inline tasks/platform-removal.rake]: yup, should have changed this to that...

File: tasks/platform-removal.rake (hunk 1 above)
Comment (melissa) [inline tasks/platform-removal.rake]: `clear` would have been the better choice here for sure

**Pull request merged**